### ISSUE TITLE
Invalid role attribute on SVGs

### ISSUE THREAD

**Issue by jvine** [sul-dlss/earthworks#634]
`<svg>`s consistently have the attribute `role="image"`, which is not correct - it should be `role="img"`

**Comment by cbeer** [comment]
This  was  fixed back in https://github.com/projectblacklight/blacklight/pull/2236, but earthworks held  an old copy in cache.
